MVP/Resources/Translations.fr.resx

```diff
@@ -0,0 +1,561 @@
+    </data>
+    <data name="contributiondetail_deleteconfirmation" xml:space="preserve">
+        <value>Êtes vous sûr de vouloir effacer cette contribution? Vous ne pouvez pas annuler cette opération.</value>
+    </data>
+    <data name="error_unexpected" xml:space="preserve">
+        <value>Une erreure est survenue. Nous avons reçu une notice de cette erreure et allons nous y atteler dès que possible.</value>
+    </data>
+    <data name="error_badrequest" xml:space="preserve">
+        <value>Votre requête n'a pas pu être traitée. Veuillez réessayer plus tard.</value>
+    </data>
+    <data name="error_servererror" xml:space="preserve">
+        <value>Votre requête n'a pas pu être traitée. Veuillez réessayer plus tard.</value>
+    </data>
+    <data name="alert_error_unauthorized" xml:space="preserve">
+        <value>Votre accréditation est expirée et n'a pas pu être rafraichie. Vous avez été déconnecté. Veuillez vous reconnecter pour continuer.</value>
+    </data>
+    <data name="clipboard_title" xml:space="preserve">
+        <value>Ça a l'air génial!</value>
+    </data>
+    <data name="clipboard_description" xml:space="preserve">
+        <value>Vous avez une URL dans votre presse-papiers. Voulez vous que nous pré-remplisson une activité basée sur cette URL?</value>
+    </data>
+    <data name="contributiondetail_notdeleted" xml:space="preserve">
+        <value>Cette contribution n'a pas pu être effacée. Peut-être elle a déjà été effacée, ou elle a étée prise en compte dans la période précédente?</value>
+    </data>
+    <data name="validation_variablefield" xml:space="preserve">
+        <value>{0} est un champ requis.</value>
+    </data>
+    <data name="warning_title_activitylocked" xml:space="preserve">
+        <value>Activité verouillée</value>
+    </data>
+    <data name="warning_description_activitylocked" xml:space="preserve">
+        <value>Cette activité a été entrée dans la periode de contribution précédente et donc elle ne peut plus être éditée ou effacée.</value>
+    </data>
+    <data name="additionaltechnologypicker_title" xml:space="preserve">
+        <value>Domaines de contributions supplémentaires</value>
+    </data>
+    <data name="error_nomvpaccount" xml:space="preserve">
+        <value>Impossible de vous authentifier avec ces informations. Êtes-vous sûr d'utiliser le compte associé à votre identifiant MVP?</value>
+    </data>
+    <data name="progress_saving" xml:space="preserve">
+        <value>Enregistrement...</value>
+    </data>
+    <data name="licenses_title" xml:space="preserve">
+        <value>Licences</value>
+    </data>
+    <data name="about_licenses_title" xml:space="preserve">
+        <value>Licences</value>
+    </data>
+    <data name="about_licenses_description" xml:space="preserve">
+        <value>Cet app utilise une série de composants sans lesquels sa création serait impossible. La liste se trouve sous les licences. Découvrez ces super logiciels.</value>
```

Comment:
Cette App repose sur les épaules de géants sans qui sa création ne pourrait être possible. Si vous voulez savoir qui ils sont, regardez les licenses pour leurs supers logiciels.

I am not sure about the intent here for "check out the licenses for their awesome pieces of software." if it's regarding other software they have produce or if it is regarding to this app.

Reply:
I like your formulation better. More literal. And from the context, it's a recommendation to see the software they have produced.